src/components/Camera/index.js

```diff
@@ -112,6 +113,7 @@ export default class Camera extends React.Component<CameraType> {
 
   componentDidMount () {
     this.webcamMounted()
+    if (this.webcam && !this.webcam.stream) return this.props.onWebcamError()
```

Comment:
Should you not pass something like `{ message: "No stream found" }`?

Reply:
I could, yes :)